src/DateRangePickerInput/makeStories.js

```diff
@@ -8,4 +8,22 @@ export default(storiesOf, {
       <DateRangePickerInput
         name={{ from: 'checkIn', to: 'checkOut' }}
       />
+    ))
+    .add('before today', () => (
+      <DateRangePickerInput
+        name="dob"
+        disabledDays={[{ before: new Date() }]}
+      />
+    ))
+    .add('all dates', () => (
```

Comment:
This story isn't needed anymore since it's the default behaviour :)

Reply:
Whoops! Updated! 👍